src/pages/DirectMessage.tsx

```diff
@@ -201,9 +251,40 @@ const DirectMessage = () => {
               <InfoOutlinedIcon />
             </IconButton>
           </Box>
-          <Divider />
+
           <Box sx={styles.chatContainer}>
             <List component="div" ref={messageRef} sx={styles.messageList}>
+              <Box
+                sx={styles.bioContainer}
+                onClick={() => navigate(`/${selectedConversation.username}`)}
+              >
+                {loading ? (
+                  <CircularProgress />
+                ) : (
+                  <>
+                    <Box sx={styles.bioDetails}>
+                      <Avatar sx={styles.avatar} />
+                    </Box>
+                    <Box sx={styles.nameContainer}>
+                      <Typography variant="h6" sx={styles.displayName}>
```

Comment:
Instead of styling this to look like a link do you think it'd be better if we use a `Link` component? Similar to what we do elsewhere in the app, for example in the post list

Reply:
I removed the hover color to make it consistent with the rest of our links